src/bitmessageqt/networkstatus.py

```diff
@@ -42,106 +49,127 @@ def __init__(self, parent=None):
+                "networkstatus",
+                "Processed %n broadcast message(s).",
+                None,
+                QtCore.QCoreApplication.CodecForTr,
+                shared.numberOfBroadcastsProcessed))
 
     def updateNumberOfPubkeysProcessed(self):
+        """"""
         self.updateNumberOfObjectsToBeSynced()
-        self.labelPubkeyCount.setText(_translate(
-            "networkstatus", "Processed %n public key(s).", None, QtCore.QCoreApplication.CodecForTr, shared.numberOfPubkeysProcessed))
+        self.labelPubkeyCount.setText(
+            _translate(
+                "networkstatus",
+                "Processed %n public key(s).",
+                None,
+                QtCore.QCoreApplication.CodecForTr,
+                shared.numberOfPubkeysProcessed))
 
     def updateNumberOfBytes(self):
         """
         This function is run every two seconds, so we divide the rate of bytes
         sent and received by 2.
         """
-        self.labelBytesRecvCount.setText(_translate(
-            "networkstatus", "Down: %1/s  Total: %2").arg(self.formatByteRate(network.stats.downloadSpeed()), self.formatBytes(network.stats.receivedBytes())))
-        self.labelBytesSentCount.setText(_translate(
-            "networkstatus", "Up: %1/s  Total: %2").arg(self.formatByteRate(network.stats.uploadSpeed()), self.formatBytes(network.stats.sentBytes())))
+        self.labelBytesRecvCount.setText(
+            _translate(
+                "networkstatus",
+                "Down: %1/s  Total: %2").arg(
+                    self.formatByteRate(network.stats.downloadSpeed()),
+                    self.formatBytes(network.stats.receivedBytes())))
+        self.labelBytesSentCount.setText(
+            _translate(
+                "networkstatus", "Up: %1/s  Total: %2").arg(
+                    self.formatByteRate(network.stats.uploadSpeed()),
+                    self.formatBytes(network.stats.sentBytes())))
 
     def updateNetworkStatusTab(self, outbound, add, destination):
+        """"""
+        # pylint: disable=too-many-branches
         if outbound:
             try:
-                c = BMConnectionPool().outboundConnections[destination]
+                BMConnectionPool().outboundConnections[destination]
             except KeyError:
                 if add:
                     return
         else:
             try:
-                c = BMConnectionPool().inboundConnections[destination]
+                BMConnectionPool().inboundConnections[destination]
             except KeyError:
                 try:
-                    c = BMConnectionPool().inboundConnections[destination.host]
+                    BMConnectionPool().inboundConnections[destination.host]
                 except KeyError:
                     if add:
                         return
 
         self.tableWidgetConnectionCount.setUpdatesEnabled(False)
         self.tableWidgetConnectionCount.setSortingEnabled(False)
-        if add:
```

Comment:
Where did this whole section go?

Reply:
If this section was ever hit, `c` would not be in scope. Since we're not seeing `NameError`s, I assume this code is never hit.

Reply:
Your assumptions are incorrect. The KeyError handlers are there to prevent adding invalid connections. Without this section the list of connected peers will remain empty forever.

Reply:
I have reverted this section and added an `undefined-variable` pylint marker on the method. If you can explain to me where `c` gets defined such that the setItem() calls don't fail I'd be keen to learn something here.